packages/rum-recorder/src/boot/recorder.entry.ts

```diff
@@ -1 +1,29 @@
-export * from '@datadog/browser-rum'
+import { defineGlobal, getGlobalObject } from '@datadog/browser-core'
+import {
+  internal_CommonContext as CommonContext,
+  internal_makeRumGlobal as makeRumGlobal,
+  internal_startRum as startRum,
+  RumGlobal,
+  RumUserConfiguration,
+} from '@datadog/browser-rum'
+
+import { startRecording } from './recorder'
+
+function startRumAndRecording(userConfiguration: RumUserConfiguration, getCommonContext: () => CommonContext) {
+  const startRumResult = startRum(userConfiguration, () => ({
+    ...getCommonContext(),
+    hasReplay: true,
+  }))
+
+  const { lifeCycle, parentContexts, configuration } = startRumResult
+  startRecording(lifeCycle, userConfiguration.applicationId, configuration, parentContexts)
+
+  return startRumResult
+}
+
+export const datadogRum = makeRumGlobal(startRumAndRecording)
+
+interface BrowserWindow extends Window {
+  DD_RUM?: RumGlobal
+}
+defineGlobal(getGlobalObject<BrowserWindow>(), 'DD_RUM', datadogRum)
```

Comment:
The `DD_RUM` global is already created in `rum.entry.ts`. We re-create and overwrite it here. This isn't pretty, but should cause any issue. Maybe we should discuss of a way to avoid creating the global in `rum.entry.ts` (a separate, internal entry point?)